docs/resources/google_compute_firewalls.md.erb

```diff
@@ -0,0 +1,80 @@
+---
+title: About the google_compute_firewalls Resource
+platform: gcp
+---
+
+# google\_compute\_firewalls
+
+Use the `google_compute_firewalls` InSpec audit resource to test properties of all GCP compute firewalls for a project.
+
+<br>
+
+## Syntax
+
+A `google_compute_firewalls` resource block collects GCP firewalls by project then tests that group.
+
+    describe google_compute_firewalls(project: 'chef-inspec-gcp') do
+      it { should exist }
+    end
+
+Use this InSpec resource to enumerate IDs then test in-depth using `google_compute_firewall`.
+
+    google_compute_firewalls(project: 'chef-inspec-gcp').firewall_names.each do |firewall_name|
+      describe google_compute_firewall(project: 'chef-inspec-gcp',  name: firewall_name) do
+        it { should exist }
+        its('kind') { should eq "compute#firewall" }
+      end
+    end
+
+<br>
+
+## Examples
+
+The following examples show how to use this InSpec audit resource.
+
+### Test that there are no more than a specified number of firewalls available for the project
+
+    describe google_compute_firewalls(project: 'chef-inspec-gcp') do
+      its('entries.count') { should be <= 100}
+    end
+
+### Test the exact number of firewalls in the project
+
+    describe google_compute_firewalls(project: 'chef-inspec-gcp') do
+      its('firewall_ids.count') { should cmp 8 }
+    end
+
+### Test that an expected firewall is available for the project
+
+    describe google_compute_firewalls(project: 'chef-inspec-gcp') do
+      its('firewall_names') { should include "my-app-firewall-rule" }
+    end
+
+### Test that a particular named rule does not exist
+
+    describe google_compute_firewalls(project: 'chef-inspec-gcp') do
+      its('firewall_names') { should_not include "default-allow-ssh" }
+    end
+
+### Test there are no firewalls for the "INGRESS" direction
+
+    describe google_compute_firewalls(project: 'chef-inspec-gcp') do
+      its('firewall_directions') { should_not include "INGRESS" }
+    end
+
+<br>
+
+## Filter Criteria
+
+This resource currently does not support any filter criteria; it will always fetch all available firewalls.
```

Comment:
Actually, you have `firewall_id`, `firewall_name`, and `firewall_direction`.  You can use any of them with `where`, as a block or as a method.

Please read https://github.com/inspec/inspec/blob/master/docs/dev/filtertable-usage.md

Reply:
thanks, likely another copy/paste oversight, will update accordingly